test/repos_conf.yml

```diff
@@ -0,0 +1,2 @@
+Factory:
+  repository: http://download.suse.de/ibs/SUSE:/Factory:/Head/standard/repodata/
```

Comment:
Um, why the internal repository? Why not public https://download.opensuse.org/tumbleweed/repo/oss/ ?
This will make it more difficult to run it outside the internal network, e.g. at Travis or public Jenkins...

BTW I'd remove that `repodata` part of the path, the repositories are normally used without it in YaST or zypper. That could be a bit confusing, I'd add it in the code.